js/components/dateRange.js

```diff
@@ -0,0 +1,239 @@
+Fliplet.FormBuilder.field('dateRange', {
+  name: 'Date range',
+  category: 'Date & time',
+  props: {
+    value: {
+      type: Object,
+      default: {
```

Comment:
Could we start the value as `null`, given the the date range JS API's getter returns either a valid date range object or `null`.